api/swagger-spec/oapi-v1.json

```diff
@@ -16987,6 +16987,10 @@
        "$ref": "v1.DeploymentCause"
       },
       "description": "extended data associated with all the causes for creating a new deployment"
+     },
```

Comment:
The change controllers update the cause and latestVersion on status when they regenerate the configs in response to triggers.